manual/README.md

```diff
@@ -146,3 +146,50 @@ The following setup assumes a Unix-like environment.
     <br/><br/>
 
 12. Apply to PoA. Please contact the Mezo team for further instructions.
+
+## Starting a node from a CometBFT snapshot
+
```

Comment:
If we reorganize as proposed in the previous comment, let's link the 'Runbooks#State sync from snapshot' section from the root's `README.md`'s [same section](https://github.com/mezo-org/validator-kit?tab=readme-ov-file#state-sync-from-snapshot). I propose to replace:
```
Mezo team does not provide any snapshots yet. It will do so for Mezo Matsnet testnet in the near future. Until then, you can ask trusted community members for a snapshot.
```
with:
```
Mezo team provides snapshots for Mezo Matsnet testnet. Please refer to [this runbook](link to 'Runbooks#State sync from snapshot' from this document) for details. Alternatively, yoiu can ask trusted community members for a snapshot.
```

Reply:
addressed in: 9dc4b0803ab5cde98258e9695ef0b95bcf6a7a60